src/sage/categories/category_with_axiom.py

```diff
@@ -1908,7 +1908,7 @@ def _base_category_class_and_axiom(cls):
             sage: CommutativeRings()._base_category_class_and_axiom
             (<class 'sage.categories.rings.Rings'>, 'Commutative')
             sage: CommutativeRings()._base_category_class_and_axiom_origin
-            'deduced by base_category_class_and_axiom'
+            'set by __classget__'
```

Comment:
do you understand how this change comes about?

Reply:
no. It may come from no longer using this category in CommutativeAlgebra,  possibly.